bin/docker_setup

```diff
@@ -53,15 +56,16 @@ Dir.chdir APP_ROOT do
   run "cp config/service_providers.localdev.yml config/service_providers.yml"
 
   puts "== Creating and migrating dev database =="
-  run "docker-compose run --rm web rake db:create"
+  run "docker-compose run --rm web bundle exec rake db:create"
   # The following pattern prevents a database reset from happening in prod.
-  run "docker-compose run --rm web rake db:environment:set"
-  run "docker-compose run --rm web rake db:reset"
-  run "docker-compose run --rm web rake db:environment:set"
+  run "docker-compose run --rm web bundle exec rake db:environment:set"
+  run "docker-compose run --rm web bundle exec rake db:reset"
+  run "docker-compose run --rm web bundle exec rake db:environment:set"
   # This populates the dev database with sample data
-  run "docker-compose run --rm web rake dev:prime"
+  run "docker-compose run --rm web bundle exec rake dev:prime"
```

Comment:
@zachmargolis I tried to collapse these commands but for some reason when I run more than one rake command at once I get `gem` errors. 🤔

Reply:
Bummer! I checked out your branch and gave it a shot locally with some stuff, couldn't figure out how to make it work. Thanks for trying!